.github/workflows/main.yml

```diff
@@ -195,7 +195,7 @@ jobs:
         --define sonar.sources=libcanard
         --define sonar.exclusions=libcanard/_canard_cavl.h
         --define sonar.cfamily.gcov.reportsPath=.
-        --define sonar.cfamily.build-wrapper-output="${{ env.BUILD_WRAPPER_OUT_DIR }}"
+        --define sonar.cfamily.compile-commands="${{ env.BUILD_WRAPPER_OUT_DIR }}/compile_commands.json"
```

Comment:
There was a warning at Sonar web console about deprecation of `sonar.cfamily.build-wrapper-output` property,
and suggestion to use `sonar.cfamily.compile-commands` instead. It worked, no warnings anymore.